addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14637,12 +14646,12 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36210"
-msgid "Prevent the 'connection lost' notification window from displaying when the PVR backend server is not available."
+msgid "Prevent the \"connection lost\" notification window from displaying when the PVR backend server is not available."
```

Comment:
Either that or you could change the setting back to all lowercase. I actually like "connection lost" more since it's less specific, i.e. if we some day change the message to "Backend connection lost" we don't need to change these settings since "connection lost" is meant to be generic. Compare e.g. `Don't show scraper errors` vs. `Don't show "Scraper failed" errors`.

It's a very minor detail though so if you don't want to bother, don't.